### PR TITLE
feat(socal-tunneller): set localPort to bind

### PR DESCRIPTION
**Pull request by mclavel**
- option to set custom local port to bind
- update tag docker img

### CODE REVIEW

File: charts/socat-tunneller/values.yaml [hunk 1]
```diff
@@ -24,3 +24,4 @@ podAnnotations: {}
 tunnel:
   host: myhost
   port: 9999
+  localPort: 9999
```
Comment (plumdog) [inline charts/socat-tunneller/values.yaml]: I think this should default to `null` to retain backwards compatibility.

File: charts/socat-tunneller/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for socat-tunneller
 name: socat-tunneller
-version: 0.1.5
+version: 0.1.6
```
Comment (plumdog) [inline charts/socat-tunneller/Chart.yaml]: I think this change is a new feature, so justifies this bumping to `0.2.0`.

Comment (mclavel) [reply charts/socat-tunneller/Chart.yaml]: ok 🎉

**Review by plumdog — APPROVED** [review]
LGTM :+1:

**Pull request merged**